pandas/tseries/tests/test_timedeltas.py

```diff
@@ -30,6 +30,17 @@ class TestTimedeltas(tm.TestCase):
     def setUp(self):
         pass
 
+    def test_contains(self):
+        # Checking for any NaT-like objects
+        # GH 13603
+        td = to_timedelta(range(5), unit='d') + pd.offsets.Hour(1)
+        for v in [pd.NaT, None, float('nan'), np.nan]:
+            self.assertFalse((v in td))
+
+        td = to_timedelta([pd.NaT])
+        for v in [pd.NaT, None, float('nan'), np.nan]:
+            self.assertTrue((v in td))
+
     def test_construction(self):
```

Comment:
can u also add tests for ``get_loc`` like:

- https://github.com/pydata/pandas/blob/master/pandas/tseries/tests/test_period.py#L3294